app/models/channel.rb

```diff
@@ -1,4 +1,8 @@
 class Channel < ActiveRecord::Base
   has_and_belongs_to_many :text_components
   belongs_to :report
+
+  def self.default(report)
+    find_by(name: "sensorstory", report: report)
```

Comment:
If we expect certain records in our database, we should use a constant like `Channel::SENSORSTORY` and `Channel::CHATFUEL`. I believe that scopes are quite handy for that, sth. like `SENSORSTORY = ~> { find_by(name: 'sensorstory') }`